jsapp/xlform/src/view.params.coffee

```diff
@@ -73,6 +75,11 @@ module.exports = do ->
           val = "#{@paramDefault}"
       else if @paramType is $configs.paramTypes.boolean
         val = evt.currentTarget.checked
+      else if @paramType is $configs.paramTypes.maxPixels
+        if evt.currentTarget.value is ' ' then evt.currentTarget.value = ''
+        val = +evt.currentTarget.value
+        if !val then val = undefined   # +'', +' ' or +'0' unsets max-pixels
```

Comment:
Together with `maxPixelsParam(…)` in `view.params.templates.coffee`, this implements a widget that accepts digits as input, treats blank input as an undefined value, and also treats a `' '` input as undefined and trims the text field to display the placeholder again.